.eslintrc

```diff
@@ -0,0 +1,109 @@
+{
+	"env": {
+		"browser": true,
+		"commonjs": true,
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"Prism": "readonly",
+	},
+	"parserOptions": {
+		"ecmaVersion": 5,
+	},
+	"root": true,
+	"rules": {
+		"comma-dangle": [
+			"warn",
+			"only-multiline",
+		],
+		"linebreak-style": [
+			"error",
+			"unix",
+		],
+		"no-console": "off",
+		"no-empty": "off",
+		"no-inner-declarations": "off",
+		"no-mixed-spaces-and-tabs": "error",
+		"no-redeclare": "off",
```

Comment:
I think that this should be set to `"error"`.

When we switch to ES6 and replace all `var`s with either `const` or `let` in the future, we will have to deal with these errors anyway, so we might as well do it now.